src/user/announcements.rst

```diff
@@ -10,6 +10,10 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+.. _dated 2022-04-23:
+
+:ref:`23 April 2022 :<dated 2022-04-23>`
```

Comment:
Let's remove the colons in the ref titles. I've been thinking about it and I think it will look better without the colons in those headers.

:ref:\`23 April 2022 <dated 2022-04-23>`